README.md

```diff
@@ -36,16 +38,19 @@
 
 Give a ⭐️ if you like this project!
 
+
 ## Contribution
 
 Contributions and issues [click](https://github.com/Suzi216/To-Do-List/issues) are most welcome !
 
+
 ## Acknowledgments
 
 - The design of this project is based from this website:[click](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
 - Inspiration
 - etc
 
+
 ## 📝 License
 
 This project is [MIT](./MIT.md) licensed.
```

Comment:
### Broken License link
- Great job adding a License to your project, but unfortunately the link is broken. You can copy the `LICENSE` file or provide a link instead